src/DisputeManager.ts

```diff
@@ -183,6 +183,9 @@ function updateAppeal(disputeId: BigInt, roundNumber: BigInt, event: EthereumEve
   appeal.settled = false
   appeal.appealDeposit = nextRound.value6
   appeal.confirmAppealDeposit = nextRound.value7
+  if (appeal.opposedRuling.gt(new BigInt(0))) {
```

Comment:
Not sure what's the recommended way of creating BNs, but in other places we seem to use `BigInt.fromI32()`